model/src/main/proto/exploration.proto

```diff
@@ -172,12 +172,16 @@ message TrainingData {
 // Structure for a single solution
 // Maps from: data/src/main/java/org/oppia/android/android/data/backends/gae/model/GaeSolution.kt
 message Solution {
-  string interaction_id = 1;
+  reserved 1; // Was a string containing the ID of the interaction containing the solution.
+
   // Flag that is true if correct_answer is the only correct answer of the question.
   bool answer_is_exclusive = 2;
 
-  // correct_answer of the State.
-  CorrectAnswer correct_answer = 3;
+  // Represented a representation of correct_answer that was specific to strings and fractions.
+  reserved 3;
+
+  // The correct answer to the state's primary question prompt.
+  InteractionObject correct_answer = 6;
```

Comment:
Seems a weird name for a correct answer (sounds like it describes the entire interaction)...

Reply:
This name is per https://github.com/oppia/oppia/blob/develop/core/domain/state_domain.py#L517. I'm not inclined to deviate from web's terminology since that can increase confusion. Do you have any suggestions?

Reply:
Wait, to be clear, I'm referring to "InteractionObject", not "correct_answer" (I'm fine with the latter). Previously this was typed as CorrectAnswer ... not sure why it changed?